db/migrate/20151106172531_remove_uniqueness_display_name_index_users.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveUniquenessDisplayNameIndexUsers < ActiveRecord::Migration
+  def change
+    remove_index :users, column: :display_name, unique: true
```

Comment:
I'm fine with multiple Jim's having the same display name. But we will always have to include the username too. Don't know of any other areas where we might not be doing that currently.